### PR TITLE
W-14147454-PM-Settings-DH-New Partner Manager Deployment Settings

### PR DESCRIPTION
**Pull request by dhirsch1**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/deploying-partner-manager.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+= Deploying Partner Manager
+
+You can deploy Anypoint Partner Manager to either Cloudhub or Anypoint Runtime Fabric.
```
Comment (kevintroller) [inline modules/ROOT/pages/deploying-partner-manager.adoc]:
```suggestion
You can deploy Anypoint Partner Manager to either CloudHub or Anypoint Runtime Fabric. 
```

Comment (kevintroller) [reply modules/ROOT/pages/deploying-partner-manager.adoc]: Based on Term Presentation Reference https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference#TermPresentationReference-C

**Review by kevintroller — DISMISSED** [review]
LGTM added minor comment

**Comment by valkyrie69** [comment]
Documented in Partner Manager documentation.

**Pull request closed without merging**